spec/models/special_offer_spec.rb

```diff
@@ -3,5 +3,8 @@
 require "rails_helper"
```

Comment:
I have some concerns about specs like this.  Are these actually useful or meaningful?  I don't think we should fall into the habit of writing spec code just so that we get the percentage up high enough to pass. Would it be more useful to test that the validation is working?